controllers/userController.ts

```diff
@@ -1,9 +1,9 @@
 import axios from "axios";
-import { User as UserProfile } from "@/types/user";
-import { Facility as BathroomProfile } from "@/types/facility";
-import { Review } from "@/types/review";
+import { User as UserProfile } from "../types/user";
+import { Facility as BathroomProfile } from "../types/facility";
 
 const port = process.env.PORT || 8081;
```

Comment:
this should probably be 3000 like the other ones. maybe move the environment variable declations into their own file so that you don't have to change them in 3 places? duplicate code makes it easy to forget changing things